### PR TITLE
@orta => [Handoff] Make Native to desktop web work.

### PR DESCRIPTION
**Pull request by alloy**

### CODE REVIEW

**Comment by alloy** [comment]
Bah, branch is named the wrong way around, but whatevs.

File: Artsy/View_Controllers/Artwork/ARArtworkViewController.m [hunk 1]
```diff
@@ -105,6 +104,14 @@ - (void)viewDidAppear:(BOOL)animated
...
     CGRect frame = self.view.frame;
     [self.view.metadataView updateConstraintsIsLandscape:CGRectGetWidth(frame) > CGRectGetHeight(frame)];
+
+    self.ar_userActivityEntity = self.artwork;
+}
+
+- (void)viewWillDisappear:(BOOL)animated;
+{
+    [super viewWillDisappear:animated];
+    [self.userActivity invalidate];
```
Comment (orta) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController.m]: doesn't this mean that you cannot go _back_ into an activity? You might just be on the iPhone and going into the "more info VC" then you'd lose context when going back

Comment (alloy) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController.m]: No I actually made that work. Every time you go back into the VC, the activity is configured from `viewDidAppear:`.

Comment (orta) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController.m]: ah yeah, I see it

**Comment by alloy** [comment]
I had some xcode scheme mess-up and wasn’t running all the tests. Some are failing, will fix that now.

**Comment by orta** [comment]
Do all of these entities get updated rather than replaced within all the loading functions for these VCs? We're making a reference to the original right away.

**Comment by orta** [comment]
If not, it's all :+1:  from my perspective

**Comment by alloy** [comment]
They get replaced, because once an activity is invalidated it can no longer become current.

**Comment by alloy** [comment]
@orta Tests should be fixed.

**Comment by orta** [comment]
Coool

**Pull request merged**